models/forms/TaskFilter.php

```diff
@@ -56,7 +64,7 @@ class TaskFilter extends Model
     public function rules()
     {
         return [
-            ['title', 'string'],
+            [['title', 'date_start', 'date_end'], 'string'],
```

Comment:
In order to assure a valid date translation add:
```
 [['date_start'], DbDateValidator::class],
 [['date_end'], DbDateValidator::class],
```
This will parse the posted date and translate the user date format into an db date format

You'll have to call `$this-validate()` at the beginning of the `query()` function in order to trigger the translation.

Reply:
You'll also have to remove

```
 [['taskAssigned', 'taskResponsible', 'own', 'status', 'overdue'], 'integer']
```